pfcli/deployment.py

```diff
@@ -343,6 +341,7 @@ def create(
         "cloud": cloud,
         "region": region,
         "total_gpus": total_gpus,
+        "infrequest_perm_check": True,
```

Comment:
Is this argument always true?

Reply:
No, it's my fault to upload this line in this PR.
Thank you for pointing it out!